### ISSUE TITLE
Shortcut key `[Period]` is somehow replaced by `F3`

### ISSUE THREAD

**Issue by Jule-** [microsoft/vscode#170503]
Type: <b>Bug</b>

1. Go to "Keyboard Shortcuts" view `Ctrl+K Ctrl+S`
2. Search for "toggle line comment"
3. Try to change default `Ctrl+Shift+[Period]` by `Ctrl+[Period]`

The created User shortcut is showing `Ctrl+F3` while in the JSON file it is correctly set to `Ctrl+[Period]` and unfortunately the shortcut you have to type in order to trigger line comment is effectively the one with `F3`.

VS Code version: Code - Insiders 1.75.0-insider (717d7d62c8b4566ab6704de931e7fe8e0649d810, 2023-01-03T08:54:14.220Z)
OS version: Linux x64 5.15.0-56-generic snap
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (12 x 3200)|
|GPU Status|2d_canvas: unavailable_software<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>opengl: disabled_off<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software<br>webgpu: disabled_off|
|Load (avg)|4, 4, 4|
|Memory (System)|15.57GB (7.20GB free)|
|Process Argv|--no-sandbox --disable-extensions --verbose --disable-gpu --sync off --wait --new-window --waitMarkerFilePath /tmp/8WnjN0IP|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details>Extensions disabled

**Comment by Jule-** [comment]
![image](https://user-images.githubusercontent.com/9716736/210399595-f1bec23e-818f-4a04-8528-434fe541cf19.png)

**Comment by Jule-** [comment]
@alexdima Hi :wave:, any news on this issue? Am I an isolated case? Can I provide you with more information? Sorry for being a little bit pushy, but this is a real productivity killer for me... :sweat_smile:  :upside_down_face: If I can do anything to help, please let me know. Thank you so much! :pray:

**Comment by hgblu-sammosna** [comment]
I have this issue too...

**Comment by Syndorik** [comment]
I'm having the same issue, it seems that it's not possible to remove the ctrl+[Period] from "breadcrumbs.focus"!

**Comment by pibenett** [comment]
Same issue here

**Comment by fguada** [comment]
Same issue for me for version 1.75. No problem with previous versions.

**Comment by Snouzy** [comment]
cmd+[Period] is not being interpreted, even changing it from settings.json. 
It's one of my best shortcut and avoid me to touch the mouse. 
Any update?

**Comment by karljstn** [comment]
Same issue here! 
Comma is interpreted as F2, and I guess there are many others.

Version is 1.75 too.

**Comment by toramanlis** [comment]
I can provide a bit of a new information.

I have the same issue with the letter "ı" in turkish keyboard. Other accents work just fine but this one is acting up. This is different from the other accents as the key types an accent letter only if it's not capitalized. If it is, it types "I". This may have something to do with the issue. Also I cannot seem to set `ctrl+[KeyI]` or `ctrl+[KeyI]` but `ctrl+alt+[KeyI]` works just fine.

I reproduced the issue with a fresh install with no extensions on a fresh user. The keyboard shortcut logs show the combination is received correctly bu fail to find the match.
[keybindings.log](https://github.com/microsoft/vscode/files/10660368/keybindings.log)

Shotcut settings:
```
[
    {
        "key": "ctrl+[KeyI]",
        "command": "workbench.action.showCommands"
    },
    {
        "key": "ctrl+alt+[KeyI]",
        "command": "workbench.action.terminal.openNativeConsole",
        "when": "!terminalFocus"
    }
]
```

VS code info:
```
Version: 1.75.0
Commit: e2816fe719a4026ffa1ee0189dc89bdfdbafb164
Date: 2023-02-01T15:29:17.766Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Linux x64 5.15.6-051506-generic
Sandboxed: No
```

OS info:
```
Operating System: Kubuntu 22.04
KDE Plasma Version: 5.24.7
KDE Frameworks Version: 5.92.0
Qt Version: 5.15.3
Kernel Version: 5.15.6-051506-generic (64-bit)
Graphics Platform: X11
Processors: 12 × AMD Ryzen 5 2600 Six-Core Processor
Memory: 15.6 GiB of RAM
Graphics Processor: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
```

**Comment by jhonore** [comment]
Same issue on version 1.75.0...
Impossible to bind something else to  "cmd+[Period]"
![image](https://user-images.githubusercontent.com/18578614/217019903-0e5be00f-1d38-4fd3-83af-7c6fcd8b0e0c.png)

It's change "[Period]"  to "F3" 
![image](https://user-images.githubusercontent.com/18578614/217020380-e7e895ca-25d6-42f1-8db6-644fb1683299.png)

Version

<img width="394" alt="image" src="https://user-images.githubusercontent.com/18578614/217020621-7b111bce-018e-4ad4-be10-49560a81949c.png">

**Comment by hugues31** [comment]
Hello,

I can confirm the issue is related to 1.75 release and may be related to specific keyboard layout (french standard azerty layout as many other in this thread).
If you installed code with snap --> `snap revert code`  until a fix pop :)

**Comment by toramanlis** [comment]
The issue seems to be fixed and published to insiders:
https://github.com/microsoft/vscode/issues/173515#issuecomment-1419012339

**Comment by OlivierMartineau** [comment]
Thank you @toramanlis, it was driving me crazy.
Fighting muscle memory is already hard, but when it is not your wish it's even harder.

**Comment by pibenett** [comment]
Thank you @toramanlis 
Do you know when will it be available for the regular (no insiders) version of VsCode please?

**Comment by toramanlis** [comment]
@pibenett no idea. I was just following another thread of this issue and saw the update

**Comment by Jule-** [comment]
Wow yeah actually fixed in insiders! :partying_face: 
Related issue: https://github.com/microsoft/vscode/issues/173325
And related PR: https://github.com/microsoft/vscode/pull/173456

No idea about the stable one, though... I am still in 1.74 I didn't know this regression had been pushed back to the stable one... But it seems the fix is on its way! (https://github.com/microsoft/vscode/pull/173585)

**Comment by Mth0158** [comment]
I have the same problem since 1.75 ... quit annoying, I hope it will be solved pronto
![image](https://user-images.githubusercontent.com/74139731/217830295-599e22b5-bc9f-4b87-9ceb-5a9b823656f5.png)

**Comment by alexdima** [comment]
I'm super sorry that I caused this regression and then I missed this issue. I was away for an extended time and I could not keep up with notifications during this time. This is fixed in Insiders and the fix is also available in the 1.75.1 stable release. (via #173325)

Many apologies for this regression!

**Comment by toramanlis** [comment]
> I'm super sorry that I caused this regression and then I missed this issue. I was away for an extended time and I could not keep up with notifications during this time. This is fixed in Insiders and the fix is also available in the 1.75.1 stable release. (via #173325)
> 
> Many apologies for this regression!

dude, it's fine. there are always issues like this. as far as i'm concerned, it's the team coordination that could be improved here. there could have been someone else available while you were away.

still, imo, you took action pretty quickly

**Comment by Jule-** [comment]
Not a problem at all, I can totally understand. 😊 Welcome back then! 😉
